### PR TITLE
NFDIV-2934 welsh nop joint js

### PR DESCRIPTION
**Pull request by itoulou**
### Change description ###

welsh content for joint js nop
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2934

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java [hunk 1]
```diff
@@ -79,7 +90,10 @@ public Map<String, Object> apply(final CaseData caseData,
             .town(town)
             .build();
 
-        if (!WELSH.equals(applicantLanguagePreference)) {
+        if (WELSH.equals(applicantLanguagePreference)) {
+            getWelshTemplateContent(templateContent, caseData);
+            ctscContactDetails.setPoBox("Blwch Post 13226");
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java:95]: how come only post box change and rest of the details stay sam for welsh. Also this not be set like this (hardcoded) it should be in config

Comment (FaisalMoJ) [reply src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java:95]: If we use `DocmosisCommonContent.getBasicDocmosisTemplateContent(LanguagePre...)` it takes care of header and footer values based on language 👍

Comment (itoulou) [reply src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java:95]: Can I change the welsh email in this method to the contactdivorce@justice.gov.uk?

Comment (FaisalMoJ) [reply src/main/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingJointJudicialSeparationContent.java]: It's already updated as part of NFDIV-3200. That ticket is in QA in progress so may be leave the values bit for now as once that PR is merged you could simply merge master? 🤔

**Review by FaisalMoJ — APPROVED** [review]

**Review by nitinprabhu — APPROVED** [review]

**Comment by amoriartyMOJ** [comment]
Closing as welsh is withdrawn.

**Pull request closed without merging**